source/nijigenerate/core/package.d

```diff
@@ -754,6 +759,18 @@ void incBeginLoop() {
                 incExit();
                 break;
 
+            case SDL_MULTIGESTURE:
```

Comment:
https://github.com/ocornut/imgui/blob/a2366f902249107f1533a85d3d1f93ec193f52f0/backends/imgui_impl_sdl2.cpp#L328
Currently Imgui does not use `SDL_FINGERUP`, `SDL_FINGERDOWN`, `SDL_MULTIGESTURE` events, which means we do not need to call `incGLBackendProcessEvent()`